### PR TITLE
Update package URL to link to Teleport docs

### LINKED ISSUE #3261: Website button on Ubuntu Software App, not working as intended
**What happened**:
I installed community teleport using `Ubuntu Software` app, and on the display screen, clicking on the `Website` button downloads "teleport-v4.2.1-linux-386-bin.tar.gz" instead of taking me to website.

**What you expected to happen**:
Lead me to gravitational's website

**How to reproduce it (as minimally and precisely as possible)**:
1. Go to https://gravitational.com/teleport/download/
1. Download "Linux 64-bit DEB"
1. Click on downloaded file
1. Default opens file on Ubuntu Software App, and click "Website" button

**Environment**:
- OS (e.g. from /etc/os-release): Ubuntu 18.04.3 LTS

Screenshot from 2020-01-10 09-45-07

### PR DESCRIPTION
**Pull request by webvictim**
Also adds an architecture hint to the package description.

Fixes #3261

### CODE REVIEW

File: build.assets/build-package.sh [hunk 1]
```diff
@@ -61,10 +61,11 @@ LINUX_CONFIG_DIR=/etc
 LINUX_DATA_DIR=/var/lib/teleport
 
 # extra package information for linux
-MAINTAINER="ops@gravitational.com"
+MAINTAINER="info@gravitational.com"
 LICENSE="Apache License 2.0"
```
Comment (benarent) [inline build.assets/build-package.sh]: Does this need to be `Apache-2.0` to show up in Ubuntu Package tool?

**Review by benarent — APPROVED** [review]

File: build.assets/build-package.sh (hunk 1 above)
Comment (webvictim) [reply build.assets/build-package.sh]: Yes, I suspect it does.

**Comment by webvictim** [comment]
Changes:

```
~/go/src/github.com/gravitational/teleport/build.assets(master*) » diff -u old.txt new.txt                                                                                                                                                                                                                         gus@antaeus
--- old.txt	2020-01-13 15:32:21.468979877 -0400
+++ new.txt	2020-01-13 15:32:09.561861976 -0400
@@ -5,15 +5,15 @@
 Install Date: (not installed)
 Group       : default
 Size        : 116653744
-License     : Apache License 2.0
+License     : Apache-2.0
 Signature   : (none)
 Source RPM  : teleport-4.2.1-1.src.rpm
-Build Date  : Mon 13 Jan 2020 15:30:58 AST
-Build Host  : d93ff37c97d2
+Build Date  : Mon 13 Jan 2020 15:29:30 AST
+Build Host  : 38cb485ec042
 Relocations : / 
-Packager    : ops@gravitational.com
+Packager    : info@gravitational.com
 Vendor      : Gravitational
-URL         : https://get.gravitational.com/teleport-v4.2.1-linux-amd64-bin.tar.gz
-Summary     : Gravitational Teleport is a gateway for managing access to clusters of Linux servers via SSH or the Kubernetes API [Open source edition]
+URL         : https://gravitational.com/teleport/docs
+Summary     : Gravitational Teleport is a gateway for managing access to clusters of Linux servers via SSH or the Kubernetes API [64-bit Open source edition]
 Description :
-Gravitational Teleport is a gateway for managing access to clusters of Linux servers via SSH or the Kubernetes API [Open source edition]
+Gravitational Teleport is a gateway for managing access to clusters of Linux servers via SSH or the Kubernetes API [64-bit Open source edition]
```

**Review by kimlisa — APPROVED** [review]
The bit information is a nice touch!

**Comment by webvictim** [comment]
These changes will take effect when new packages are built for the next release of Teleport.

**Pull request merged**